src/SeleniumLibrary/keywords/browsermanagement.py

```diff
@@ -371,6 +371,8 @@ def create_webdriver(
         `Close All Browsers` keyword is used. See `Switch Browser` for an
         example.
         """
+        if kwargs is None:
+            kwargs = {}
         if not isinstance(kwargs, dict):
             raise RuntimeError("kwargs must be a dictionary.")
```

Comment:
These two line you could removed. 
Robot Framework will fail, if you give anything that can not be converted to dict.
At least if we are not covering every bad things people could do from Python.

Reply:
For these I would prefer to keep them there. Relying on another piece code, even a single entry point type path, is something I prefer not to do.